README.md

```diff
@@ -8,55 +8,53 @@ non-unified and formatted and unformatted files are supported.
 
 *libecl* is mainly developed on *Linux* and *OS X*, in addition there
 is a portability layer which ensures that most of the functionality is
-available on *Windows*. The main functionality is written in C, and
+available on *Windows*. The main functionality is written in C/C++, and
 should typically be linked in in other compiled programs. *libecl* was
 initially developed as part of the [Ensemble Reservoir
 Tool](http://github.com/Statoil/ert), other applications using
 *libecl* are the reservoir simulator flow and Resinsight from the [OPM
 project](http://github.com/OPM/).
 
-In addition to the C code there are Python wrappers which make most of
-the *libecl* functionality available from Python. For small interactive
-scripts, forward models e.t.c. this is recommended way to use *libecl*
-functionality.
+In addition to the compiled C/C++ code there are Python wrappers which make most
+of the *libecl* functionality available from Python. For small interactive
+scripts, forward models e.t.c. this is the recommended way to use *libecl*
+functionality. You decide wether to build include the Python wrappers when
+configuring the `cmake` build - pass the option `-DENABLE_PYTHON=ON` to enable
+Python wrappers, by default the Python wrappers are *not* included.
 
-
-### Compiling the C code ###
+### Alternative 1: Building without Python ###
 *libecl* uses CMake as build system:
 
 ```bash
-git clone https://github.com/Statoil/libecl.git
+git clone https://github.com/Statoil/libecl
 cd libecl
 mkdir build
 cd build
-cmake ..
+cmake .. 
 make
+make install
 ```
 If you intend to develop and change *libecl* you should build the tests
 by passing `-DBUILD_TESTS=ON` and run the tests with `ctest`.
 
 
-
-### Compiling the Python code ###
-
-Python is not a compiled language, but there is a basic "build system"
-which does a basic Python syntax check and configures some files to
-correctly set up the interaction between the Python classes and the
-shared libraries built from the C code.
-
-You need to install some Python requirements before the Python code
-will work:
+### Alternative 2: Building with Python ###
+To build *libecl* with Python wrappers you just pass the option
+`-DENABLE_PYTHON=ON` when running `cmake`. In addition you need to install some
+dependencies first:
 
     sudo pip install -r requirements.txt
```

Comment:
if we leave this line here, it's unclear when it has to be run (I know it's obvious if you know what you are doing, but let's make it as easy as possible).
I'd suggest to include it in the script below after the `cd libecl`

Reply:
OK - updated.